packages/google_maps_flutter/google_maps_flutter_ios/example/ios13/ios/RunnerUITests/RunnerUITests.m

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+static const CGFloat kWaitTime = 15;
```

Comment:
This should be much longer (maybe 60), since Flutter policy is to not have timeouts in tests, to avoid flake. It's really inconvenient to have no timeouts with the XCUITest APIs, so a very long value is our best option.